docs/operating-scylla/nodetool-commands/resetlocalschema.rst

```diff
@@ -0,0 +1,27 @@
+============================
+Nodetool resetlocalschema
+============================
+
+This command clears all locally stored schema information and resets the schema 
+to the initial state. It helps you resolve schema disagreement.
```

Comment:
What does "initial" state mean in this context?
Is this operation relevant after consistent schema updates with Raft?
@denesb ?

Reply:
We need to ask @tgrabiec.

Reply:
"resets the schema to the initial state" is wrong. The schema is not changed, it's reloaded from disk, simulating alter table events. It's still illegal to modify schema sstables while the node is running and use resetlocalschema to make the node reflect that - this attempt will fail, possibly by crashing the node.

Works the same with raft.